### PR TITLE
fix: UI edge cases of handling test errors

### PR DESCRIPTION
**Pull request by elit-altum**
Fixes some UI edge cases:
1. Marks the run status as `error` whenever an error is generated by Newman, previously it used to keep showing `running` even when the run ended.
2. Does not update the CPU/Memory stats when the run gets paused.

### CODE REVIEW

File: dashboard/frontend/actions/index.js [hunk 1]
```diff
@@ -72,10 +75,19 @@ const mountSockets = (store) => {
...
+            // mark run as errored
+            run.setInterrupted();
+        }
     });
 
     socket.on('run-stats', (data) => {
         const run = store.find(data.id);
+
+        // ignore the run stats if it is paused
+        if(run.isPaused()) return;
```
Comment (coditva) [inline dashboard/frontend/actions/index.js]: Why do we want to ignore the run stats if the run is paused?

Comment (elit-altum) [reply dashboard/frontend/actions/index.js]: Wouldn't that be ideal? I mean the run should not be doing anything right? 
Previously it used to keep updating the stats even when paused, should I keep it like that?

**Review by coditva — APPROVED** [review]
